README.md

```diff
@@ -53,7 +53,8 @@ The following is the corresponding `torchdata` versions and supported Python ver
 
 | `torch`              | `torchdata`        | `python`          |
 | -------------------- | ------------------ | ----------------- |
-| `master` / `nightly` | `main` / `nightly` | `>=3.8`, `<=3.10` |
+| `master` / `nightly` | `main` / `nightly` | `>=3.8`, `<=3.11` |
+| `2.0.0`              | `0.6.0`            | `>=3.8`, `<=3.11` |
```

Comment:
Ohh, I am sorry. We should not touch compatibility table until the official release is provided. My bad. Let's revert the last commit

Reply:
Reverting

Reply:
I think nightly should still be `<=3.11`? What do you think?

Reply:
Yeah